weed/filer/elastic/v7/elastic_store.go

```diff
@@ -73,6 +66,25 @@ func (store *ElasticStore) Initialize(configuration weed_util.Configuration, pre
 	}
 	return nil
 }
+
+func (store *ElasticStore) initialize(configuration weed_util.Configuration, prefix string) (options []elastic.ClientOptionFunc) {
```

Comment:
this PR is merged. but usually `Initialize()` handles configuration object, and `initialize()` do the actual initialization work. 

This PR has the opposite way. Maybe you can help to refactor this in a separate PR?

Reply:
ok.